package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "config.js",
   "scripts": {
+    "test:k6": "k6 run k6_tests/rps100k.js; k6 run k6_tests/rps200k.js; k6 run k6_tests/rps300k.js; k6 run k6_tests/rps400k.js; k6 run k6_tests/rps500k.js; k6 run k6_tests/rps600k.js; k6 run k6_tests/rps700k.js; k6 run k6_tests/rps800k.js; k6 run k6_tests/rps900k.js;",
```

Comment:
very comprehensive! i think you can also collect and run all of your k6 tests from a single file, but this definitely works too